pkg/metallb/metallb.go

```diff
@@ -336,6 +337,37 @@ func (builder *Builder) WithOptions(options ...AdditionalOptions) *Builder {
 	return builder
 }
 
+// WithFRRConfigAlwaysBlock adds specific routes to block from being advertised to the FRR nodes.
+func (builder *Builder) WithFRRConfigAlwaysBlock(prefixes []string) *Builder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Adding prefixes to block in the metallb.io object %s", builder.Definition.Name)
+
+	if len(prefixes) < 1 {
+		glog.V(100).Infof("the frrConfiguration prefixes list is empty")
+
+		builder.errorMsg = "can not accept empty prefix list for the metallb alwaysBlock mode"
+	}
+
+	// Validate CIDR prefixes
+	for _, prefix := range prefixes {
+		if _, _, err := net.ParseCIDR(prefix); err != nil {
+			glog.V(100).Infof("the frrConfiguration prefix %s is not a valid CIDR", prefix)
+			builder.errorMsg = fmt.Sprintf("The prefix %s is not a valid CIDR", prefix)
```

Comment:
error message should start with lowercase

Reply:
missed that one :) updated